### DISCUSSION TITLE
Best practice of using Vercel Git Integration within Nx mono repo

### DISCUSSION THREAD

**Discussion by iam-yan** [vercel/vercel#4522] [Help] [answered]
My use case:
- I use nx mono repo to manage and share code between nextjs projects.
- I use Git Integration to auto deploy.

So how could I deploy each project to production on demand?

**Comment by coetry** [comment, marked as answer]
Hi @juuyan we have plans to support monorepros with the Git integration, but currently don't have a way to do that.

**Reply by iam-yan** [reply]
Great! May I ask when might these features be released? 😃

**Reply by coetry** [reply]
We don't have a tangible date, but hopefully very soon! We've been thinking about it for a long time and would love to make it available for our customers as soon as possible.

**Reply by iam-yan** [reply]
Thanks ! Looking forward.

**Reply by mcsdevv** [reply]
Hey @juuyan, this is now available! You can read more about monorepo support here: https://vercel.com/blog/monorepos

**Reply by iam-yan** [reply]
@mcsdevv COOL! Thanks for notifying me!
